docs/using-cinder-csi-plugin.md

```diff
@@ -41,6 +41,13 @@ Encode your ```$CLOUD_CONFIG``` file content using base64.
 Update ```cloud.conf``` configuration in ```manifests/cinder-csi-plugin/csi-secret-cinderplugin.yaml``` file
 by using the result of the above command.
 
+> NOTE: openstack supports get metadata data for new deployed vm through both config drive and metadata service methods. If you need modify the search order, please add Metadata section in your cloud.conf which is openstack cloud definition. For example, below sample makes CSI pod first search config drive then metadata service. If you don't provide the value, then default search order `configDrive, metadataService` will be used.
```

Comment:
Maybe you didn't see my previous suggestion, which I think is more clear and succinct

> NOTE: In OpenStack, the compute instance uses either config drive or metadata service to retrieve instance-specific data. As the cluster administrator, you are able to config the order in the cloud config file for cinder-csi-plugin, the default configuration is as follows: